README.md

```diff
@@ -34,15 +49,18 @@ default of 8.4) you have to set this attribute as a node override:
 Attributes
 -----------
 
-`['pg-multi']['master']` : sets the IP address that defines the master node
+`['pg-multi']['master']` : sets the IP address that defines the master node (Required)
```

Comment:
Are these correct? `attributes/default.rb` contains this (master vs. master_ip, slave vs. slave_ip):
```default['pg-multi']['master_ip'] = ''
default['pg-multi']['slave_ip'] = []```